vnsi.c

```diff
@@ -148,9 +148,12 @@ bool cPluginVNSIServer::SetupParse(const char *Name, const char *Value)
     TimeshiftBufferSize = atoi(Value);
   else if (!strcasecmp(Name, CONFNAME_TIMESHIFTBUFFERFILESIZE))
     TimeshiftBufferFileSize = atoi(Value);
-  else if (!strcasecmp(Name, CONFNAME_TIMESHIFTBUFFERDIR))
+  else if (!strcasecmp(Name, CONFNAME_TIMESHIFTBUFFERDIR)) {
```

Comment:
On 2016/07/05 19:12, Rainer Hochecker <notifications@github.com> wrote:
> mind putting curly brackets on new line?

Ok, done.  (I should really add a Kodi profile to my editor.)